types/material-ui/index.d.ts

```diff
@@ -869,7 +880,7 @@ declare namespace __MaterialUI {
     namespace DatePicker {
         interface DatePickerProps {
             // <TextField/> is the element that get the 'other' properties
-            DateTimeFormat?: Intl.DateTimeFormat;
+            DateTimeFormat?: typeof Intl.DateTimeFormat;
```

Comment:
What are the actual restrictions on this type? Could this just be `new () => Intl.DateTimeFormat`?